benchmarks/Gemfile

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+ruby "2.7.1" # Keep updated to latest.
+
+gem "benchmark-ips"
```

Comment:
I've used both `benchmark` (part of stdlib) and `benchmark-ips` and I'm not sure if `benchmark-ips` is still the Ruby best practice. I went with it mostly because it makes a nicer report.

Thoughts?

Reply:
Ruby's built-in `benchmark` has results that [look like this](https://ruby-doc.org/stdlib-2.5.0/libdoc/benchmark/rdoc/Benchmark.html):

```
              user     system      total        real
foo:      0.960000   0.000000   0.960000 (  0.957966)
bar:      0.960000   0.000000   0.960000 (  0.960423)
baz:      0.950000   0.000000   0.950000 (  0.954864)
```

i.e. no ordering by speed, no "1.3x slower," and no "±" to indicate standard deviation.

Reply:
I'm with you on `benchmark-ips` then, specifically I find the ordering by speed and the "1.3x slower" to be very helpful